src/App.tsx

```diff
@@ -10,9 +32,17 @@ export const App = () => {
 
       <div className="section">
         <div className="container">
-          <h1 className="title">Home Page</h1>
-          <h1 className="title">Page not found</h1>
-          <PeoplePage />
+          <Routes>
+            <Route path="/" element={<h1 className="title">Home Page</h1>} />
+            <Route path="/home" element={<Navigate to="/" replace />} />
+            <Route path="/people" element={<PeoplePage />}>
+              <Route path="/people/:slug" element={<PeoplePage />} />
+            </Route>
```

Comment:
The nested Route inside the '/people' path is incorrect. The nested Route should be placed inside the element of the parent Route, not as a direct child of the Route component. Consider restructuring this to properly nest the '/people/:slug' route under the '/people' route.